### DISCUSSION TITLE
How do I encrypt data?

### DISCUSSION THREAD

**Discussion by Tomic-Riedel** [supabase/supabase#3881] [Questions]
Hi!

I have the following problem:
I want to encrypt a column in my table because I am very privacy conscious. 

However, I have a problem: If I add a new row via the SQL editor and then frame the entry of titles (the column that should be encrypted) with crypt(), it works great. But now I want to add a Row from my Flutter App, which also works great. The only problem is that the column is not encrypted. Does anyone know how I can solve this?

Here is my Flutter code that should add a new Row where one cell is encrypted:
```dart
class SupabaseHabitHandler {
  Future<PostgrestResponse> addHabitToSupabase(
      {required String id, required String title}) async {
    return await supabase.from("user_entries").insert({
      "user_id": id,
      "title": title,
    }).execute();
  }
}
```

**Comment by alphanumericaracters** [comment]
Well...... Im ussing Supabase to test an app on Kodular, but i send critical info like Phone_Number, ID_Number (Like a Passaport number), Geolocalitation coordinates. And i have to use curl to POST and GET to connect with the API.

And i want to know how make more secure that connection.

It´s not for payments, just for appointments at an emergency veterinarian, but I must keep that data safe.

If anyone can help me....
